tests/resources/generator/hodgkin_huxley_squid_axon_model_1952/model.external.py

```diff
@@ -89,20 +92,20 @@ def compute_computed_constants(variables):
 
 
 def compute_rates(voi, states, rates, variables, external_variable):
-    variables[0] = external_variable(voi, states, rates, variables, 0)
+    variables[0] = external_variable(voi, states, variables, 0)
     variables[11] = 0.1*(variables[0]+25.0)/(exp((variables[0]+25.0)/10.0)-1.0)
     variables[12] = 4.0*exp(variables[0]/18.0)
     rates[0] = variables[11]*(1.0-states[0])-variables[12]*states[0]
     variables[13] = 0.07*exp(variables[0]/20.0)
     variables[14] = 1.0/(exp((variables[0]+30.0)/10.0)+1.0)
     rates[1] = variables[13]*(1.0-states[1])-variables[14]*states[1]
-    variables[17] = external_variable(voi, states, rates, variables, 17)
+    variables[17] = external_variable(voi, states, variables, 17)
```

Comment:
This external variable also doesn't feature in the `compute_variables` function.

Reply:
Done in commit e133304.